common/scripts/babel-conditional-preprocess.js

```diff
@@ -0,0 +1,80 @@
+Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
Please add a top-level comment explaining what this preprocessing step does, and where it is used.